docs/domains/DataAccess/services/DataAcquisitionService/openapi.yml

```
@@ -973,6 +973,186 @@ paths:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LocationOrgPostModel'
+        "400":
+          description: Bad Request
+        "404":
+          description: Facility Not Found
+        "500":
+          description: Internal Server Error
+    put:
+      tags:
+      - location-org-config
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)